src/System.Net.WebSockets.Client/src/System/Net/WebSockets/WinRTWebSocket.cs

```diff
@@ -105,6 +114,20 @@ public override string SubProtocol
                 websocketControl.SupportedProtocols.Add(subProtocol);
             }
 
+            if (MessageWebSocketClientCertificateSupported)
```

Comment:
I was recommending throwing an exception when trying to get or set `ClientCertificates` on `ClientWebSocketOptions`. 
I think it's better to be able to discover this immediately and root-cause it instead of trying to figure out why, after setting all properties, the app doesn't use client certificate authentication.